uber/models.py

```diff
@@ -1669,6 +1669,7 @@ class ApprovedEmail(MagModel):
 
 class Email(MagModel):
     fk_id   = Column(UUID, nullable=True)
+    email_id = Column(UnicodeText)
```

Comment:
That sounds good. I hadn't heard of that convention before, but I think it will work.